deploy/local_single_user_deploy.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+# Variables ____________________________________________________________________________________________________________
+# Define some colors
+ERR='\033[1;31m\033[41m'
+INFO='\033[1,36m\033[42m'
+DEBUG='\033[0;33m'
+NC='\033[0m' # No Color
+NUM_KEEPERS=5
```

Comment:
you may consider setting the default value of `NUM_KEEPERS` based on how many lines there are in `${CONF_DIR}/hosts_keeper`. say I have three lines of `127.0.0.1` or `localhost` in it means I want to have three keepers. this makes it consistent with distributed setup where we have one keeper host per line.

Reply:
For the local deployment I am not using hosts files. Should I?

Reply:
> For the local deployment I am not using hosts files. Should I?

You are right. We only use `NUM_KEEPERS`.